__mocks__/react-native-cycling-sensors.ts

```diff
@@ -1,4 +1,22 @@
-class Sensor {
+import {SensorProps} from '../src/components/Sensor';
```

Comment:
This is just the bare essentials for a dummy mock. We can improve it as we build out more tests.